desktop-environment.el

```diff
@@ -152,6 +152,10 @@ The shell command should let the user interactively select the
 portion of the screen."
   :type 'string)
 
+(defcustom desktop-environment-screenshot-delay-argument " -d %d"
```

Comment:
can you please use `--delay` instead of `-d`?

Reply:
Should be resolved as of 9d2a1aa